assessment_software/src/app/profile-avatar/profile-avatar-component.css

```diff
@@ -2,19 +2,19 @@
     @apply font-sans;
 }
 .profile-container{
-    @apply w-max inline-flex flex justify-end float-right;
+    @apply inline-flex float-right;
```

Comment:
Try to do it using only flex? avoid using float and make it your last resort until you really get comfortable with flex and grid.

Reply:
`inline-flex justify-end` might work but i am not sure. [LINK](https://tailwindcss.com/docs/justify-content#end)